tasks/show_linters_issues/show_linters_issues.py

```diff
@@ -69,4 +69,8 @@ def show_linters_issues(
     display = display_result(lints_filtered_by_team)
     if filter_team:
         print(f"Results of running '{filter_linters}' linters on {filter_team} team owned files:\n")
-    print(display)
+        print(display)
+    else:
+        print(display)
+        print("Number of errors per team:")
+        print(count_lints_per_team(merged_results, filter_linters))
```

Comment:
🐒 Nitpick :
Shouldn't we format the print output instead of simply printing out the dictionary ?
like
```
 - team-name1 : 101010 
 - team-name2 : 10110
  ...
```

Reply:
Agreed, fixed in [fa53a5e](https://github.com/DataDog/datadog-agent/pull/19579/commits/fa53a5e0b375cbf314b884c6f2264f38d88d3910) !